### ISSUE TITLE
Proposal: Change native lib instead

### ISSUE THREAD

**Issue by fregante** [sindresorhus/ts-extras#53]
I'm hesitant to use this library as is because to a new developer this looks unnecessary:

```js
arrayIncludes(arr, value)
```

and will likely attempt to change it to:

```js
arr.includes(value)
```

which may even pass the tests and continue being less strict.

What do you think about offering **additional entry points** to change the global lib instead? I do this pattern to make some Promise types stricter for example: https://github.com/microsoft/TypeScript/issues/45602#issuecomment-934427206

**Comment by sindresorhus** [comment]
I'm open to it as long as it's clearly documented that it should not be used in reusable projects.

**Comment by fregante** [comment]
That's right. However I'm not entirely sure it can be loaded "as a whole" because *maybe* not all types are desired + it would cause more breaking releases. 

Having to import specific "fixes" would be best, like Object.entries and Array#includes

**Comment by sindresorhus** [comment]
Seems verbose and annoying to import individual types for this though. Maybe we start by exporting all and see if it causes any problems? We would document that those types do not follow semver and need to be pinned.

**Comment by fregante** [comment]
It's literally less verbose than the current situation though 😃 

Import object.entries once per project vs once per file. 

I'm ok with the global to start though

**Comment by privatenumber** [comment]
Note, overwriting `typeRoots` will [remove the default behavior of automatically detecting `node_module/@types`](https://www.typescriptlang.org/tsconfig#typeRoots).

**Comment by fregante** [comment]
Found a WIP library that does this by default:

- https://github.com/total-typescript/ts-reset#make-includes-on-as-const-arrays-less-strict
